### PR TITLE
Small improvements to Task Center - task #1942

### PR DESCRIPTION
**Pull request by windymilla**
1. Remove references to version (meaning sister sites) from Task Center interface and create upgrade script to drop column from tasks table
2. Add sort indicator arrow to heading of the column currently used for sorting in the tasks list 

Sandbox at: https://www.pgdp.org/~windymilla/c.branch/task-tasks

### CODE REVIEW

**Comment by windymilla** [comment]
Upgrade scripts are new to me, so please let me know if what I have done is either wrong, or not the preferred method. Thanks

File: SETUP/upgrade/19/20230627_alter_tasks.php [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+
+echo "Delete task_version column from tasks table\n";
+$sql = "
+    ALTER TABLE tasks
+        DROP COLUMN task_version
+";
+
+echo "$sql\n";
+
+mysqli_query(DPDatabase::get_connection(), $sql) or die(mysqli_error(DPDatabase::get_connection()));
```
Comment (chrismiceli) [inline SETUP/upgrade/19/20230627_alter_tasks.php:17]: You'll want to update the initial create schema as well. https://github.com/DistributedProofreaders/dproofreaders/blob/aae0d560d23a110d3489eee7d490decd4bb3f9fb/SETUP/db_schema.sql#L546

Comment (windymilla) [reply SETUP/upgrade/19/20230627_alter_tasks.php:17]: Thanks @chrismiceli

**Review by cpeel — APPROVED** [review]
Code and upgrade script look good!

**Review by 70ray — APPROVED** [review]

**Pull request merged**